integrated_channels/integrated_channel/exporters/learner_data.py

```diff
@@ -557,7 +492,8 @@ def _collect_certificate_data(self, enterprise_enrollment):
 
         Args:
             enterprise_enrollment (EnterpriseCourseEnrollment): the enterprise enrollment record for which we need to
-            collect completion/grade data
+            collect completion/grade data,
+            channel_name: labeled for relevant integrated channel this is being called for to enhance logging.
```

Comment:
When do we want `channel_name` to be a part of the doc string? I noticed above that it wasn't included.

Reply:
I updated params when I added them- existing params were left alone.